perfkitbenchmarker/scripts/database_scripts/plot_scatter_points.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generates gnuplot info file.
+
+Given the TPS data file, returns a gnuplot info file.
+"""
+
+
+CHART_TITLE = "Sysbench TPS"
+
+X_LABEL = 'thread count'
+Y_LABEL = 'tps'
+
+# This variable controls the verticle lines inside the chart.
+#   0 means no vertical lines.
+Y_HEIGHT = '3000'
+Y_TICS = '100'
+X_INTERVAL = '1200'
+ITERATIONS = '9'
+
+
+class gnuplot_info():
+
+  def __init__(self, gnuplot_data_filename):
+    self.output_dir = "./charts"
+    self.output_chart = self.output_dir + "/innodb_pages.png"
```

Comment:
i would use os.path.join to join paths as it is more clear what your intent is. also don't have to worry about ensuring your forward slashes are correct